### PR TITLE
Fix timezone in data view

### PR DESCRIPTION
**Pull request by ct-goranpalibrk**
Fix date-time format in all views and grids and Description tabs.

### CODE REVIEW

**Comment by lorthirk** [comment]
Can you please link relevant issues?

**Comment by ct-goranpalibrk** [comment]
This PR is for issue #935

**Comment by Coduz** [comment]
Uhm it seems that with this PR all the dates. Am I seeing right?

**Comment by ct-goranpalibrk** [comment]
Yes you are right. But @LeoNerdoG  has something for discussion and he will write it in Monday morning..

**Comment by LeoNerdoG** [comment]
Hi @ct-goranpalibrk @lorthirk @ct-goranpalibrk,
last week Goran and I had some problems with displaying correct time in the _**Description**_ tab. Where I had GMT time (instead of GMT + 02:00), Goran had the proper time and we could not figure out why (the only difference was that I was running Docker container, whereas Goran was not).
Today, when I again tested this branch, this issue does not manifest itself any more and all the other dates are also OK, so this PR can be merged, as it succesfully resolves issue #935.

**Comment by lorthirk** [comment]
What environment was Goran using? If running from Eclipse, this could be related to #49.

Anyway, let me review the code before merging, since I expected a change only in Data view according to the issue, not in every view. While it's good to be consistent I would like to check that everything is in place first.

**Comment by ct-goranpalibrk** [comment]
@lorthirk I am using eclipse and GWT plugin, and at my side everything was okay, and Leonardo says that everything is ok now also at his side, so I think that is no problems with environment.

**Comment by lorthirk** [comment]
Yes, and I think it's because of the enforcing of the UTC time zone being removed.

**Comment by ct-goranpalibrk** [comment]
Yes, I removed that. If that's not okay, I will revert that.

File: console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/util/DateUtils.java [hunk 1]
```diff
@@ -13,15 +13,14 @@
...
 
-import com.google.gwt.i18n.client.TimeZone;
-
 import com.google.gwt.i18n.client.DateTimeFormat;
 
 public class DateUtils {
 
 //    private static final ConsoleMessages MSGS = GWT.create(ConsoleMessages.class);
 
     private static final String NONE = "None";
```
Comment (Coduz) [inline console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/util/DateUtils.java]: Can you please localize this by using the Console Messages?

**Comment by ct-goranpalibrk** [comment]
@lorthirk About your proposal for changing format in `EEE dd MMM yyyy HH:mm:ss Z`, that format will produce result like this 
onez

I think that is little bit more user-friendly this format `EEE dd MMM yyyy HH:mm:ss ZZZZ` and it displays time like this: 
fourz
But if you don't thinks so, I will keep this your suggestion.

**Comment by lorthirk** [comment]
I think we can avoid the `GMT`. It doesn't add any valuable information to the timezone indication, and there are already a lot of informations packed in a small string, so sparing some of them should lead to a more clear information. Please use `Z`.

**Comment by lorthirk** [comment]
After a quick chat with @LeoNerdoG we agreed to use `EEE dd MMM yyyy HH:mm:ss ZZZZ` as the format.

**Comment by ct-goranpalibrk** [comment]
Format EE dd MMM yyyy HH:mm:ss ZZZZ is now added.

File: console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/util/DateUtils.java (hunk 1 above)
Comment (lorthirk) [reply console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/util/DateUtils.java]: @ct-goranpalibrk,

@Coduz meant to localize line 22 (`NONE`), not 23 (`FORMAT`). I fixed this in 551cde5.

**Comment by LeoNerdoG** [comment]
@lorthirk I have verified that this PR successfully resolves issue with time zones and all the changes, that have been made, are OK. This can be merged when ready.

**Pull request merged**